runtime/browser/ui/gtk/select_file_dialog_gtk2.cc

```diff
@@ -0,0 +1,547 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
```

Comment:
I think it's also worth adding a comment pointing out the file in Chrome (./chrome/browser/ui/libgtk2ui/select_file_dialog_impl_gtk2.cc , right?) also it'd be nice to try to move this to such a place in upstream so that we wouldn't have to copy it (like ./ui/shell_dialogs or ./ui/libgtk2ui).